src/ledger/GeneralizedLedgerEntry.cpp

```diff
@@ -288,10 +288,10 @@ GeneralizedLedgerKey::toString() const
         return xdr::xdr_to_string(ledgerKey());
```

Comment:
Perhaps this function should be implemented as a `cereal_override`? What are the advantages and disadvantages? Same question for `GeneralizedLedgerEntry::toString`.

Reply:
This `xdr::xdr_to_string` on L288 is removed in the next commit! (I first removed `xdr_printer`, and I removed `xdr::xdr_to_string` in the next commit.)

`GeneralizedLedgerEntry` doesn’t seem to be an XDR. I’m still learning about Cereal and `cereal_override`, but can I still use `cereal_override` for that?

Reply:
In general you can use the `cereal_override` mechanism for anything. But upon revisiting this I remembered that those `toString` functions aren't actually used (as far as I know). I provided them for debugging purposes, and they were useful when I was building this. Given that, I'd say there is no reason to spend the effort to change it to `cereal_override`.

Reply:
Got it! I'll leave it as is.